server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -98,6 +99,16 @@ const accountMutations = {
           }
         }
 
+        models.Activity.create({
```

Comment:
Let's `await` for them
```suggestion
        await models.Activity.create({
```

Reply:
@Betree : The reason I didn't add these awaits is that in my local when I added `await` for `editAccountSetting` I noticed that there's a certain lag that's happening in the front-end side. Is there a specific reason why we need to add `await` here. As far as I see we don't care for an immediate update of the log. 

[awiat problem.webm](https://user-images.githubusercontent.com/12435965/189729826-06423403-3a59-452e-97ea-d02357ad6caf.webm)

Reply:
This only effects, `editAccountSetting` and `editAccountFeeStructure` where we have a `sequelize.transaction`. So every change within needs to pass the `transaction` object which I mistakenly didn't pass. I've corrected this and now everything works perfectly with `await`. 👍🏽